### PR TITLE
Ivar trim tool: fix bug

### PR DESCRIPTION
**Pull request by gallardoalba**
FOR CONTRIBUTOR:
* [x] - I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [x] - License permits unrestricted use (educational + commercial)
* [ ] - This PR adds a new tool or tool collection
* [ ] - This PR updates an existing tool or tool collection
* [x] - This PR does something else (explain below)

This PR fixes a bug in the prepare_amplicon_info.py script, and includes and additional test.

### CODE REVIEW

**Comment by gallardoalba** [comment]
Could you have a look at it @pvanheus? I think that it is correct.

File: tools/ivar/ivar_trim.xml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<tool id="ivar_trim" name="ivar trim" version="@VERSION@+galaxy2">
+<tool id="ivar_trim" name="ivar trim" version="@VERSION@+galaxy3">
```
Comment (bernt-matthias) [inline tools/ivar/ivar_trim.xml:1]: Please add a recent profile.

Comment (bgruening) [reply tools/ivar/ivar_trim.xml]: Done :)

**Comment by wm75** [comment]
@gallardoalba Can you explain what you are trying to fix here?
The amplicon info file is supposed to consist of simple lines of tab-separated primer names.

**Comment by gallardoalba** [comment]
> @gallardoalba Can you explain what you are trying to fix here?
> The amplicon info file is supposed to consist of simple lines of tab-separated primer names.

We receipt an error report from an user, and  she provided this dataset as _amplicon info file_ https://github.com/galaxyproject/tools-iuc/blob/4e95773df83abeaf68088136d3fcff3e26f6aaea/tools/ivar/test-data/covid19/amplicon.bed. The prepare_amplicon_info.py script allows to convert this kind of dataset in a list of tab-separated primer names, but I'm not sure if the provided dataset has the correct format.

**Comment by wm75** [comment]
I see. I checked the user's input file and this is just not of the format expected by the tool.
Closing this as not concerning the wrapper.

**Pull request closed without merging**